src/containers/Review/assignment/AssignAll.tsx

```diff
@@ -30,16 +33,25 @@ const AssignAll: React.FC<AssignAllProps> = ({ assignments, setReviewerForAll, c
       <Label className="uppercase-label" content={strings.ASSIGN_ALL_TO} />
       <Dropdown
         className="reviewer-dropdown"
-        options={options}
-        placeholder={strings.ASSIGN_ALL_PLACEHOLDER}
-        value={selected}
-        clearable
-        onChange={(_, { value }) => {
-          const reviewerId = value !== '' ? options[value as number].reviewerId : 0
-          setReviewerForAll(reviewerId)
-          setSelected(value as number | string)
-        }}
-      />
+        text={
+          options.find(({ value }) => value === selected)?.text || strings.ASSIGNMENT_NOT_ASSIGNED
+        }
+        scrolling
+      >
+        <Dropdown.Menu>
+          <Dropdown.Header icon="tags" content={strings.ASSIGNMENT_NOT_ASSIGNED} />
+          <Dropdown.Divider />
+          {options.map((assignee) => (
+            <Dropdown.Item
+              {...assignee}
+              onClick={(_, { value }) => {
+                setReviewerForAll(value as number)
+                setSelected(value as number | string)
+              }}
```

Comment:
This is the same as `onChange` was previously doing, but didn't work with this new shape of Dropdown